client/services/code_preprocessors/PythonCodePreprocessorService.js

```diff
@@ -74,7 +76,11 @@ tie.factory('PythonCodePreprocessorService', [
     // given code snippet. We keep these as instance methods (instead of
     // classmethods) because that makes line number tracking less complicated,
     // since classmethods will need an extra '@classmethod' line.
-    var _wrapCodeIntoClass = function(code, wrapperClassName) {
+    var _wrapCodeIntoClass = function(
+        code, wrapperClassName, shouldProcessInternalFunctionReferences) {
+      if (shouldProcessInternalFunctionReferences) {
+        code = _addClassWrappingToStudentHelperFunctions(code);
```

Comment:
This will probably cause us issues if the code is a wrapper class vs. an instance of a class, so I'll mark that, too.